lib/Jit/options.cpp

```diff
@@ -48,9 +48,7 @@ void freeStringConfigValue(ICorJitInfo *CorInfo, UTF16CharT *Value) {
   return CorInfo->freeStringConfigValue((wchar_t *)Value);
 }
 
-Options::Options(LLILCJitContext *Context) : Context(Context) {}
-
-void Options::initialize() {
+JitOptions::JitOptions(LLILCJitContext &Context) : Context(Context) {
```

Comment:
Maybe I was a bit unclear.  The calls that do the queries to fill out the options need Context.  And by design they're not particularly tied to the constructor.  Today they're all called in the constructor but that's not required.  I could change it so that all getters need to be passed a Context but that would complicate any refactoring where they were called outside the constructor.  So It's needed.